server/controllers/lecture_booking-controller.js

```diff
@@ -0,0 +1,52 @@
+const knex = require("../db");
+const moment = require("moment");
+
+//Insert new booking of a lecture
+
+exports.newBooking = async (req, res) => {
```

Comment:
This method should be either in `lecture-controller` or `user-controller` since the `lecture_booking` table just represents a relationship.

Reply:
I thought that since we need to insert a new record exactly in the table lecture_booking i put this method here.

Reply:
It's not a stupid assumption, but if you think about it, a tuple from the `lecture_booking` table is not an object, it just represents the relationship between two objects.